src/lib/viewers/doc/DocFirstPreloader.js

```diff
@@ -67,6 +67,10 @@ class DocFirstPreloader extends EventEmitter {
 
     preloadedImages = {};
 
+    thumbnailsOpen = false;
```

Comment:
Does this cause in stuttering on the page?

Reply:
no it doesn't cause any issues